map_model/src/make/merge_intersections.rs

```diff
@@ -15,10 +15,12 @@ pub fn merge_short_roads(map: &mut RawMap, consolidate_all: bool) -> BTreeSet<No
     for r in map.roads.keys() {
         queue.push_back(*r);
 
-        // TODO Remove after improving this heuristic.
+        // TODO Enable after improving this heuristic.
+        /*
```

Comment:
Which lint was this? Searching for "if false" isn't turning anything up... I prefer disabled code to comments; otherwise some methods might get marked as unused, or the disabled code will start to drift as other APIs evolve.

Reply:
Clippy denies expressions such as `false && ...` ([clippy::logic_bug](https://rust-lang.github.io/rust-clippy/master/index.html#logic_bug)). I agree that your approach is more bit-rot resistant. I'll change it to disable the warning here instead.

Reply:
Done.